protocol/network/7-properties.md

```diff
@@ -1,16 +1,105 @@
+7 Properties
+============
 
-# IPFS Protocol on the Wire
+## 7.1 Communication Model - Streams
 
-Authors: [Juan Benet](http://github.com/jbenet)
+The Network layer handles all the problems of connecting to a peer, and exposes
+simple bidirectional streams. Users can both open a new stream
+(`NewStream()`) and register a stream handler (`SetStreamHandler`). The user
+is then free to implement whatever wire messaging protocol she desires. This
+makes it easy to build peer-to-peer protocols, as the complexities of
+connectivity, multi-transport support, flow control, and so on, are handled.
```

Comment:
Not sure if I like Bind or Mount the better, but I do have a suggestion for proto.Dial :)

`n.Dial(PeerInfo, {protocol: <if any preference, otherwise use the first it can connect from the list of multiaddrs available and those that are supported by us>})`